simulation/common/helpers.py

```diff
@@ -758,11 +774,11 @@ def simple_plot_graph(data, title, visible=True):
         plt.show()
 
 
-def calculate_race_completion_time(path_length, cumulative_distances):
+def calculate_completion_index(path_length, cumulative_distances):
```

Comment:
This function appears a bit more complicated than it needs to be. Maybe simplify?

Reply:
Definitely, I'll work on that.